politeiad/plugins/pi/pi.go

```diff
@@ -71,6 +99,14 @@ var (
 		"A-z", "0-9", "&", ".", ",", ":", ";", "-", " ", "@", "+", "#",
 		"/", "(", ")", "!", "?", "\"", "'",
 	}
+
+	// SettingProposalDomains contains the default proposal domains.
+	SettingProposalDomains = []string{
```

Comment:
Map since its only used to search.

Reply:
We don't have any way to pass in a map as a plugin setting.  Having it as a map in the pi plugin is fine, but we need to keep it an array in the API.